document/core/syntax/instructions.rst

```diff
@@ -196,12 +200,12 @@ SIMD instructions provide basic operations over :ref:`values <syntax-value>` of
 
 .. math::
    \begin{array}{llcl}
-   \production{ishape} & \X{ixx} &::=&
+   \production{ishape} & \ishape &::=&
      \K{i8x16} ~|~ \K{i16x8} ~|~ \K{i32x4} ~|~ \K{i64x2} \\
-   \production{fshape} & \X{fxx} &::=&
+   \production{fshape} & \fshape &::=&
      \K{f32x4} ~|~ \K{f64x2} \\
-   \production{vshape} & \X{vxx} &::=&
-     \X{ixx} ~|~ \X{fxx} \\
+   \production{shape} & \shape &::=&
+     \ishape ~|~ \fshape \\
    \production{lane index} & \laneidx &::=& \byte \\
```

Comment:
Just noted this. This should rather be `\i8`, since `byte` is only intended for uninterpreted binary data. In fact, it would make more sense to make it `\i4`, such that the value range is maintained by construction (with `\Bu4` in the binary format, such that bad indices become a decoding error).

Reply:
I fixed this in #346 based on your comment in another PR, so I'll leave this here, and rebase after that is merged.